lib/bundler/lazy_specification.rb

```diff
@@ -41,6 +41,8 @@ def to_lock
         out = "    #{name} (#{version}-#{platform})\n"
       end
 
+      out = out.dup
```

Comment:
Could we set an `out = String.new` variable at the beginning of the method instead of calling `dup` here?